tox.ini

```diff
@@ -6,7 +6,9 @@ envlist = py27-{1.7,1.8,1.9}
 usedevelop = True
 setenv =
   XUNIT_FILE=nosetests-{envname}.xml
-  NOSE_COVER_MIN_PERCENTAGE=100 
+  1.7: NOSE_COVER_MIN_PERCENTAGE=90
+  1.8: NOSE_COVER_MIN_PERCENTAGE=90
+  1.9: NOSE_COVER_MIN_PERCENTAGE=80
```

Comment:
This is a bit hacky, but and it would probably make sense to combine the coverage reports in the travis.yml file at a later date. I'm happy to add that to this library.